package/libpiksi/libpiksi/src/sbp_rx.c

```diff
@@ -105,8 +143,14 @@ int sbp_rx_attach(sbp_rx_ctx_t *ctx, pk_loop_t *pk_loop)
   assert(ctx != NULL);
   assert(pk_loop != NULL);
 
+  // Must assign loop anew or assign the same loop
```

Comment:
comment style

Reply:
fixed, realized I didn't need the new structure member